fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1144,6 +1170,26 @@ public Builder setInitialDelayMillis(long initialDelayMillis) {
             return this;
         }
 
+        /**
+         * Get the minimum time between successful progress logs when building across transactions.
+         * @return the minimum time between successful progress logs.
+         * Negative will not log at all, 0 will log after every commit.
```

Comment:
This line probably belongs in the main javadoc body rather than after the return param (I think).